src/views/forms/TriggersForm/TriggersForm.tsx

```diff
@@ -31,19 +46,39 @@ export const TriggersForm = ({
   onViewOrdersClick,
 }: ElementProps) => {
   const stringGetter = useStringGetter();
+  const dispatch = useDispatch();
 
-  const { asset, entryPrice, stepSizeDecimals, tickSizeDecimals, oraclePrice } =
+  const { placeTriggerOrders } = useSubaccount();
+  const isAccountViewOnly = useSelector(calculateIsAccountViewOnly);
+
+  const { asset, entryPrice, size, stepSizeDecimals, tickSizeDecimals, oraclePrice } =
     useSelector(getPositionDetails(marketId)) || {};
+
+  const { differingOrderSizes, inputErrors, inputSize, isEditingExistingOrder, existsLimitOrder } =
+    useTriggerOrdersFormInputs({
+      marketId,
+      positionSize: size?.current ?? null,
+      stopLossOrder: stopLossOrders.length === 1 ? stopLossOrders[0] : undefined,
+      takeProfitOrder: takeProfitOrders.length === 1 ? takeProfitOrders[0] : undefined,
+    });
+
   const symbol = asset?.id ?? '';
+  const multipleTakeProfitOrders = takeProfitOrders.length > 1;
+  const multipleStopLossOrders = stopLossOrders.length > 1;
 
-  const isDisabled = false; // TODO: CT-625 Update based on whether values are populated based on abacus
-  const isEditingExistingTriggers = stopLossOrders.length > 0 || takeProfitOrders.length > 0;
+  const hasInputErrors = inputErrors?.some(
+    (error: ValidationError) => error.type !== ErrorType.warning
+  );
+  const inputAlert = getTradeInputAlert({
+    abacusInputErrors: inputErrors ?? [],
+    stringGetter,
+    stepSizeDecimals,
+    tickSizeDecimals,
+  });
 
   // The triggers form does not support editing multiple stop loss or take profit orders - so if both have
   // multiple, we hide the triggers button CTA
-  const existsEditableOrCreatableOrders = !(
-    stopLossOrders.length > 1 && takeProfitOrders.length > 1
-  );
+  const existsEditableOrCreatableOrders = !(multipleTakeProfitOrders && multipleStopLossOrders);
 
   const priceInfo = (
```

Comment:
actually re price info -- maybe we need to care about ticksize
<img width="590" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/8a00022a-707f-47ac-af2c-ebb99094fd07">